src/components/Wizard/BlueprintWizard.js

```diff
@@ -61,6 +68,10 @@ const BlueprintWizard = (props) => {
     useSelector((state) => selectAllImageTypes(state));
   const imageTypes = getImageTypes();
 
+  const getBlueprintNames = () =>
+    useSelector((state) => selectAllBlueprintNames(state));
+  const blueprintNames = getBlueprintNames();
```

Comment:
A shorter way of doing this is just 
`const blueprintNames = useSelector((state) => selectAllBlueprintNames(state));`

Reply:
The image types call above is excessive and I should fix it.